modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointHistory.java

```diff
@@ -174,9 +175,10 @@ public WALPointer firstCheckpointPointer() {
      * is not finished yet.
      *
      * @param entry Entry to add.
+     * @param cacheGrpStates
```

Comment:
I believe it makes sense to add some comment why using cacheGrpStates from entry is a bad idea(to prevent further mistakes)

Reply:
I doubt that someone will reuse this method, and if they do, they can find current fix I guess.